android/kotlin_guide.md

```diff
@@ -0,0 +1,49 @@
+# Kotlin guide
+This guide is designed to help you get started with [Kotlin], the language we write most of our code in, and to introduce you to some intermediate topics.
+- [Getting Started](#getting-started)
+- [Staying Informed](#staying-informed)
+- [Intermediate Topics](#intermediate-topics)
+
+## Getting Started
+There are multiple ways to get started with Kotlin, depending on how you like to learn. If you like:
+- Reading documentation, the official Kotlin docs are good: start at [Basic Syntax] and work your way through the pages in the side bar
+- Learning while coding, try the [Kotlin Koans] online or in the IDE
+- Courses, try the official [Kotlin for Android Developers][course] course (note: none of our team members have tried this yet)
+- Reading books, try [Kotlin in Action]
```

Comment:
Maybe mention the bookshelf for SF employees?  Also probably worth including the Big Nerd Ranch book.

Reply:
fwiw, I'm trying to reduce choice here to keep it simple for people. I phrased it to emphasize Kotlin in Action.